org.mixedrealitytoolkit.spatialmanipulation/Solvers/RadialViewDisplayInitializeHelper.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clause
+
+using UnityEngine;
+
+namespace MixedReality.Toolkit.SpatialManipulation
+{
+    /// <summary>
+    /// Positions a game object with a RadialView component at the center of the view,
+    /// maintaining a distance equal to the average of the MinDistance and MaxDistance
+    /// properties of the RadialView. This prevents the object from rapidly approaching the
+    /// viewer and ensures it appears upright.
+    /// </summary>
+    [RequireComponent(typeof(RadialView))]
+    public class RadialViewDisplayInitializeHelper : MonoBehaviour
+    {
+        private RadialView radialView;
+
+        protected void Awake()
+        {
+            radialView = GetComponent<RadialView>();
+        }
+
+        private void OnEnable()
+        {
+            var distance = (radialView.MinDistance + radialView.MaxDistance) / 2.0f;
```

Comment:
I wonder if it would make more sense to include this as a configurable behavior on RadialView instead of adding an additional script. I.E. adding a serialized field CenterInitialPosition, and if that is true then making this adjustment.

That would also allow SolverHandler.TransformTarget to be used in place of Camera.main.transform

Reply:
That is a possibility. I was trying to mess as little as possible with existing code. :)

Reply:
@AMollis do you have a preference here?